hnn_core/params_default.py

```diff
@@ -388,27 +383,6 @@ def _secs_L2Pyr():
         'basal_2': [[-50, 715, 0], [-156, 609, 0]],
         'basal_3': [[-50, 715, 0], [56, 609, 0]],
     }
-    # increased by 70% for human
-    sec_lens = {  # microns
-        'soma': 22.1,
-        'apical_trunk': 59.5,
-        'apical_oblique': 340,
-        'apical_1': 306,
-        'apical_tuft': 238,
-        'basal_1': 85,
-        'basal_2': 255,
-        'basal_3': 255
-    }
-    sec_diams = {  # microns
-        'soma': 23.4,
-        'apical_trunk': 4.25,
-        'apical_oblique': 3.91,
-        'apical_1': 4.08,
-        'apical_tuft': 3.4,
-        'basal_1': 4.25,
-        'basal_2': 2.72,
-        'basal_3': 2.72
-    }
```

Comment:
this should be defined in the param file

Reply:
Is it currently defined there?

Reply:
yep, and extracted here:

https://github.com/jonescompneurolab/hnn-core/blob/2f1da11e8f6fdd17f469ce2f5f1376526741d84d/hnn_core/cells_default.py#L21

Reply:
Just to be clear: you mean that the section length and diameters are still read from param-files? You're just removing the hard-coded values?

Reply:
yep, exactly

Reply:
Out of curiosity, why do we now assume that the length and width morphology parameters are defined in the .param file but not the section points?

Reply:
Just because historically they have not been defined in the .param files ...

Reply:
I know this is an old complaint, but at some point we should standardize what is specified in the param file and what is hard coded. Since length and width are an important part of morphology, it would make the most sense for them to be stored and implemented through a similar codepath as the section points. For a later PR though....

Reply:
what makes the most sense to me is to write up a reader for the morphological file and store the geometry of the default cells in that file format. Then, finally deprecate params related to the geometry in the param file.